examples/tv-app/android/java/ContentAppCommandDelegate.cpp

```diff
@@ -41,7 +41,7 @@ using LaunchResponseType         = chip::app::Clusters::ContentLauncher::Command
 using PlaybackResponseType       = chip::app::Clusters::MediaPlayback::Commands::PlaybackResponse::Type;
 using NavigateTargetResponseType = chip::app::Clusters::TargetNavigator::Commands::NavigateTargetResponse::Type;
 using GetSetupPINResponseType    = chip::app::Clusters::AccountLogin::Commands::GetSetupPINResponse::Type;
-using Status                     = Protocols::InteractionModel::Status;
+using Status                     = chip::Protocols::InteractionModel::Status;
```

Comment:
why the extra `chip` here? We are in the chip namespace and there are no other `Protocols` right?
I notice that the other using are using it, so is it just a matter of keeping the original style?

Reply:
Mass find and replace side effect